umbra-js/src/utils/utils.ts

```diff
@@ -283,9 +283,9 @@ function getResolutionInstance(provider: EthersProvider) {
   const networkName = provider.network.name === 'homestead' ? 'mainnet' : provider.network.name;
   return new Resolution({
     sourceConfig: {
-      cns: {
-        provider: Eip1993Factories.fromEthersProvider(provider),
-        network: networkName as 'mainnet' | 'rinkeby',
+      uns: {
```

Comment:
Looks like the tests don't run since this config change was not updated in the tests. So  I made this same config change, but the unstoppable tests in `cns.test.ts` now fail:

- `gets the public keys associated with a CNS address` fails with `ResolutionError: Domain udtestdev-msolomon.crypto is not configured`
- `throws when the user has not set their stealth keys` fails with `Domain udtestdev--c38898.crypto is not configured` (which is not the expected error message).

Tests in `utils.test.ts` fail similarly. Those were the original domains I registered for testing, but it seems they are no longer configured? I don't quite understand what changed—was everything migrated to a new system and this should still work, or was the old system discarded and/or my names expired?

Reply:
I didn't run the tests at first, but i tested it by sending and receiving a stealth payment and it worked. I modified the config but tests still fail (and testing it by sending a payment still works), it's possible that the old system was discarded or the names expired.. i don't know honestly. Maybe a test with a newly registered ud domain be worthwhile (with it's spending and viewing keys).